lib/hsa/unpinned_copy_engine.cpp

```diff
@@ -525,3 +522,23 @@ void UnpinnedCopyEngine::CopyPeerToPeer(void* dst, hsa_agent_t dstAgent, const v
         hsa_signal_wait_acquire(_completionSignal2[i], HSA_SIGNAL_CONDITION_LT, 1, UINT64_MAX, HSA_WAIT_STATE_ACTIVE);
     }
 }
+
+
+bool UnpinnedCopyEngine::GetPointerInfo(const void *ptr)
```

Comment:
This function should be renamed because it's not getting the pointer info but instead checking whether a pointer is locked.

Reply:
I'm Ok with the proposed changes, could use some help if possible to actually implement them.